components/hab/static/template_plan.sh

```diff
@@ -0,0 +1,12 @@
+pkg_name={{ pkg_name }}
+pkg_origin={{ pkg_origin }}
+pkg_version=0.0.1
+pkg_description="The package description"
```

Comment:
I can agree with that. :) I'm thinking something like what one finds in a standard Linux config file, a bunch of options commented out with their default values (if any) and a brief comment about what they do.